source/atlas-cli-local-cloud.txt

```diff
@@ -37,9 +37,9 @@ actions:
    manage that deployment with the {+atlas-cli+}.
 
 You can use only ``atlas deployments`` {+atlas-cli+} commands with 
-local |service| deployments. You can use all {+atlas-cli+} commands, 
-including ``atlas deployments`` commands,  with cloud |service| 
-deployments.
+local |service| deployments and cloud deployments. You can use all 
+other {+atlas-cli+} commands, including ``atlas deployments`` 
+commands,  with cloud |service| deployments.
```

Comment:
I don't think this statement makes sense- how can it be that they can use **only** atlas deployments with both types, but can also use all with cloud? I think maybe you meant to say this?
"You can use ``atlas deployments`` {+atlas-cli+} commands with..." (remove the "only" on line 39